tasks/bundle.js

```diff
@@ -54,3 +58,13 @@ constants.partialBundlePaths.forEach(function(pathObj) {
         pathToMinBundle: pathObj.distMin
     });
 });
+
+// "Browserify" the locales
+var localeGlob = path.join(constants.pathToLib, 'locale-*.json');
+glob(localeGlob, function(err, files) {
+    files.forEach(function(file) {
+        var outName = 'plotly-' + path.basename(file).replace(/.json$/, '.js');
+        var outPath = path.join(constants.pathToDist, outName);
+        wrapLocale(file, outPath);
+    });
+});
```

Comment:
@etpinard what do you think about this solution?
- Turned the locales into json files
- Explicitly loaded just `'en'` and `'en-US'` into the built bundles (ie I got rid of `locales.js` so that if/when we add more locales they will not automatically go into all the bundles) but we'll still put new locales into `lib/` for use if you make your own bundle.
- Built separate minified js files for each locale, so if you load via a script tag (from CDN or script) you can first load plotly.js and then in the next script tag load the locale you want.

Reply:
> Turned the locales into json files

Requiring JSON files isn't ideal as described in https://github.com/plotly/plotly.js/pull/2195#discussion_r155848922 

I'd vote for converting those `lib/locale-???` files into plain js files. 

> so that if/when we add more locales they will not automatically go into all the bundles) but we'll still put new locales into lib/ for use if you make your own bundle.

great :ok_hand: 

> Built separate minified js files for each locale, so if you load via a script tag (from CDN or script) you can first load plotly.js and then in the next script tag load the locale you want.

Lovely :heart: and by the looks of it out [push-to-cdn](https://github.com/plotly/streambed/blob/master/bin/update_plotlyjs_cdn.sh) script should _just work_<sup>tm</sup>.